site/src/scaffold/scripts/form-submit.ts

```diff
@@ -0,0 +1,49 @@
+export function handleMockFormSubmit() {
+	document.querySelectorAll('form')?.forEach(form => {
+		form.addEventListener('submit', function (e) {
```

Comment:
window.addEventListener with third parameter `{ intercept: true}`

Reply:
It doesn't seem like `intercept` is one of the available option properties for addEventListener?

Reply:
🤦‍♂️🤦‍♂️ capture

Reply:
Added